server/src/main/java/org/tbbtalent/server/model/es/CandidateEs.java

```diff
@@ -289,7 +290,15 @@ public void copy(Candidate candidate, TextExtracter textExtracter) {
         List<CandidateEducation> educations = candidate.getCandidateEducations();
         if (educations != null) {
             for (CandidateEducation education : educations) {
-                final String text = education.getCourseName();
+                String text = null;
+                final EducationMajor educationMajor = education.getEducationMajor();
+                if (educationMajor != null) {
+                    text = educationMajor.getName();
+                }
+                final String courseName = education.getCourseName();
+                if (courseName != null) {
+                    text = text == null ? courseName : text + " " + courseName;
+                }
```

Comment:
I see why you're doing this. But it means the elastic search query will still fail in some cases because the education major field search is specified as a term query which will look for exact keyword matches.

We could change the way the search is setup in SavedSearchServiceImpl/computeElasticQuery(..) or fix the underlying candidate data before re-building the elastic search index. 

What do you think?

Reply:
I tested this and it worked

Reply:
Note we'll need to rebuild the elastic search index before testing.

But even so it will only work in specific cases - i.e. where the education major is null and the course name matches one of the education major drop-down values.

Reply:
I'll take another look, but I did rebuild the ES and tested it and convinced myself that it worked. But maybe I missed something.

Reply:
@sadatmalik I have tested and checked in a new version - can you take a look?